### DISCUSSION TITLE
Great job getting those Windows domains!

### DISCUSSION THREAD

**Discussion by ghost** [schrebra/Windows.10.DNS.Block.List#2] [Ideas]
Thanks for researching this.  One of the big domain blocks that needs to be investigated is Microsoft.  They do a lot of analytics, tracking, and ad serving, but practically no DNS blocking site calls them out for this.  Your list is a huge step in the right direction.

Would you be interested in trimming your list to a text file so that it focuses on ad serving, adware, analytics, and tracking?

**Comment by schrebra** [comment]
Thank you for the feedback and positive comments. The list is created from mostly Microsoft's privacy websites and were forced to publicly publish what every app phones home to. Probably GDPR related. From what I remember the way Microsoft categorized the domains were application and Windows version specific. As you can imagine there was a lot of overlapping and duplicate domains. There isn't a good way to know what is an Ad domain or tracking. You could take a look at my other list that has other awesome Windows 10 blocking hosts.
https://github.com/schrebra/combined.list
Also added .txt versions as requested.
